functions.js

```diff
@@ -1 +1,77 @@
-// YOUR CODE HERE
+// 1.) ### `maxOrMin(num1, num2, max)`
+function maxOrMin(num1, num2, max){
+  if(max) {
+    return Math.max(num1,num2);
+  } else {
+    return Math.min(num1,num2);
+  }
+}
+maxOrMin(556,665,(1>2));
+maxOrMin(556,665,(1<2));
+//not sure if I should be using these
+
```

Comment:
Got it. I wasn't sure if it was supposed to be something that evaluates to true or false, or if I just plug it in like you just mentioned.